sys/shell/commands/sc_openwsn.c

```diff
@@ -0,0 +1,578 @@
+    }
+
+    num = atoi(argv[2]);
+
+    if (!strcmp(argv[1], "add")) {
+        code = IANA_6TOP_CMD_ADD;
+        if (!msf_candidateAddCellList(celllist_add, num)) {
+            puts("Error: can't add that many cells");
+            return -1;
+        }
+    }
+    else if (!strcmp(argv[1], "rmv")) {
+        code = IANA_6TOP_CMD_DELETE;
+        if (!msf_candidateRemoveCellList(celllist_delete, &neigh, num,
+                                         cell_options)) {
+            puts("Error: can't remove the specifide cells");
+            return -1;
+        }
+    }
+    else if (!strcmp(argv[1], "rel")) {
+        code = IANA_6TOP_CMD_RELOCATE;
+        if (!schedule_getCellsToBeRelocated(&neigh, celllist_delete) || \
+            !msf_candidateAddCellList(celllist_add, num)) {
+            puts("Error: failed to get cells to relocate");
+            return -1;
+        }
+    }
+    else {
+        puts("Error: unknown 6top command");
+        return -1;
+    }
+
+    int ret = sixtop_request(code, &neigh, num, cell_options, celllist_add,
+                             celllist_delete, IANA_6TISCH_SFID_MSF,
+                             0, 0);
+    if (ret) {
+        puts("Error: 6top request failed");
+        return -1;
+    }
+
+    return 0;
+}
+
+static int _6top_cmd(int argc, char **argv)
+{
+    if (argc < 2) {
+        _print_6top_usage();
+        return 1;
+    }
+
+    if (!strcmp(argv[1], "clear")) {
+
+        open_addr_t neigh;
+
+        if (argc == 3) {
+            neigh.type = ADDR_64B;
+            size_t len = netif_addr_from_str(argv[2], neigh.addr_64b);
+            if (len == 0) {
+                puts("Error: invalid address");
+                return -1;
+            }
+        }
+        else {
+            if (IS_USED(MODULE_OPENWSN_IPV6)) {
+                if (!icmpv6rpl_getPreferredParentEui64(&neigh)) {
+                    puts("Error: no preferred parent");
+                    return -1;
+                }
+            }
+            else {
+                puts("Error: a neighbour address must be supplied");
+                return -1;
+            }
+        }
+        int ret = sixtop_request(IANA_6TOP_CMD_CLEAR, &neigh, 0, 0, NULL,
+                                 NULL, IANA_6TISCH_SFID_MSF, 0, 0);
+        if (ret) {
+            puts("Error: 6top request failed");
+            return -1;
+        }
+    }
+
+    if (!strcmp(argv[1], "add") || !strcmp(argv[1], "rmv") || \
+        !strcmp(argv[1], "rel")) {
+        return _6top_manage_cmd(argc, argv);
+    }
+
+    _print_6top_usage();
+    return 1;
+}
+
+static void _print_queue_usage(void)
+{
+    puts("Usage:");
+    puts("\tqueue list: show all Openqueue entries");
+    puts("\tqueue rmv <creator>: remove all entries from <creator> in queue");
+}
+
+static int _queue_cmd(int argc, char **argv)
+{
+    if (argc < 2) {
+        _print_queue_usage();
+        return 1;
+    }
+
+    if (!strcmp(argv[1], "list")) {
+        bool queue = 1;
```

Comment:
Maybe rename that variable `empty_queue`. Otherwise the check below looks weird:
```
if (queue) {
   puts("Queue is empty");
}
```

Reply:
Also use `true` and `false` (since it's a boolean).